mlops/common/naming_utils.py

```diff
@@ -63,3 +63,19 @@ def generate_run_name(build_id: str):
     build = os.environ.get("BUILD_BUILDID")
 
     return f"run_{build}"
+
+
+def generate_environment_name(environment_name, environment_version):
```

Comment:
please add the variable type to align with the convention in the other methods. looks like both will be of type `str`

Reply:
updated signature